src/Components/Results/Results.css

```diff
@@ -278,13 +235,6 @@
   border-radius: 0.75rem;
 }
```

Comment:
I left these two `result-program-more-info-wrapper` and `result-program-more-info-button` styles since these affect the `More Info` button in the long term benefits tab.